go/gas-oracle/flags/flags.go

```diff
@@ -32,6 +43,16 @@ var (
 		Usage:  "Hardcoded tx.gasPrice, not setting it uses gas estimation",
 		EnvVar: "GAS_PRICE_ORACLE_TRANSACTION_GAS_PRICE",
 	}
+	EnableL1BaseFeeFlag = cli.BoolFlag{
```

Comment:
Is there ever a reason why we would disable one of these services?

Reply:
We may want to disable one of the services in an extreme case where there is a bug in one of them, similarly to the way the batch submitter is configured.

Reply:
It also lets it be backwards compatible, we can turn off the L1 base fee updating and it should still work on the old `OVM_GasPriceOracle`